src/lib/quoteEngine.js

```diff
@@ -26,25 +33,24 @@ const { formatEther } = ethers.utils;
  *
  * It sorts pools by the base price and then takes full capacity from the pools with
  * the lower prices until it reaches desired coverAmount
```

Comment:
(docs) need to update this as its no longer accurate

Reply:
fixed